t/sql-embed.neon

```diff
@@ -5,9 +5,9 @@ EXEC SQL CONNECT TO :DatabaseName;
 EXEC SQL DROP TABLE IF EXISTS user;
 EXEC SQL CREATE TABLE user(name STRING PRIMARY KEY, password STRING);
 
-LET name: String := "name"
+LET user_name: String := "name"
```

Comment:
Don't change this existing test here. Instead, add a few new lines to the bottom of this file that tests the use of an underscore in a parameter value.

Reply:
Added a new test below the current one.  I only used one parameter with an underscore; should I have used two or more?

Reply:
One is sufficient.